recipe/meta.yaml

```diff
@@ -77,7 +78,7 @@ outputs:
         - {{ pin_subpackage('gap-core', exact=True) }}
     test:
       commands:
-        - echo "TODO - test a package here"
+        - echo 'if LoadPackage("smallgrp") = fail then QUIT_GAP(1); else QUIT_GAP(0); fi;' | gap
```

Comment:
Sorry, the code snippet was proposed in part by ChaptGPT (since I don't speak GAP.)

> But this is a bad idea: first, it is redundant as smallgrp is always loaded anyway. Secondly it will just hang in case of a problem with loading GAP and/or any if the autoload packages.

We just want to test something not entirely trivial that is not just calling `gap -h`.  Also, it did not hang for me when I put a typo into `smallgrp`.

Reply:
Your example produces `Could not read file "QuitGap(LoadPackage("smallgrp"));".` and then hangs.

Reply:
That sound as if either the `-c` was missing or the input was used with a GAP version <= 4.10.x, as `-c` was added in GAP 4.11.0

Reply:
Yes, it was 4.10.0 that I had locally.